tests/Feature/SnmpTraps/ApcPduOutletTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * ApcPduOutletTest.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ */
+
+namespace LibreNMS\Tests\Feature\SnmpTraps;
+
+use App\Models\Device;
+use App\Models\Ipv4Address;
+use LibreNMS\Snmptrap\Dispatcher;
+use LibreNMS\Snmptrap\Trap;
+use LibreNMS\Tests\LaravelTestCase;
+
+class ApcPduOutletTest extends LaravelTestCase
+{
+    public function testOutletOff()
+    {
+        $device = factory(Device::class)->create();
+        $ipv4 = factory(Ipv4Address::class)->make();
```

Comment:
$ipv4 is unused in all 3 tests, delete it ;)

Reply:
Done.